src/refinement-list/refinement-list.ts

```diff
@@ -151,4 +152,19 @@ export class NgAisRefinementList extends BaseWidget {
       this.state.refine(item.value);
     }
   }
+
+  public updateState = (state: RefinementListState): void => {
```

Comment:
Copied from [this](https://github.com/algolia/vue-instantsearch/blob/c023ba0fafbb32d556db2895aafec6c3742a1380/src/components/RefinementList.vue#L181-L191).